packages/component-model-manuscript/entities/manuscript/index.js

```diff
@@ -177,6 +177,15 @@ class Manuscript extends BaseModel {
     return manuscripts
   }
 
+  async $afterInsert() {
+    await new AuditLog({
+      action: 'CREATED',
+      objectId: this.id,
+      objectType: 'manuscript',
+      value: JSON.stringify(this.toJSON()),
```

Comment:
Just ran this locally - and there seems to be a lot of information that is redundant going into the database :

```
| {"meta":{"title":"","articleType":"","subjects":[]},"opposedSeniorEditorsReason":"","opposedReviewingEditorsReason":"","opposedReviewersReason":"","files":[],"coverLetter":"","status":"INITIAL","previouslyDiscussed":null,"previouslySubmitted":[],"cosubmission":[],"submitterSignature":"","disclosureConsent":false,"teams":[],"createdBy":"e61db86c-5233-4249-b384-2f0cc2546dd2","id":"a1a3b661-fbae-438f-9157-0ef82dc29867","created":"2019-07-05T07:32:36.609Z","updated":"2019-07-05T07:32:36.609Z"} |
```
I'm wondering that we should just log the `id` and the `createdBy`

Reply:
Maybe a deep diff and omit values that remain the same?

Reply:
^ yeah this :)